specificatie/openapi.yaml

```diff
@@ -1445,13 +1500,26 @@ components:
       enum:
       - "huwelijk"
       - "geregistreerd_partnerschap"
-    Geboortedatum:
+    GeboorteBasis:
       type: "object"
       description: |
                     Gegevens over de geboorte van respectievelijk de persoon, de ouder, de echtgenoot/geregistreerd partner, de eerdere echtgenoot/geregistreerd partner of het kind.
                     * **datum** : Datum waarop de persoon is geboren
-                    * **land** : Land waar de persoon is geboren
-                    * **plaats** : De plaats waar de persoon is geboren. Voor een plaats buiten Nederland is gemeentecode=1999 (RNI) en gemeentenaam de buitenlandse plaatsnaam of aanduiding.
       properties:
         datum:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/DatumOnvolledig"
+    GeboortedatumInOnderzoek:
```

Comment:
Ook hier twijfel ik over. Wat voegt dit toe in een zoekresultaat?

Reply:
als ik me goed herinner hebben we besloten inOnderzoek er wel in te laten